generate-event.go

```diff
@@ -365,3 +402,176 @@ var scheduleEvent = `{
+       },
+       {
+        "name": "MIME-Version",
+        "value": "1.0"
+       },
+       {
+        "name": "From",
+        "value": "Jane Doe <janedoe@example.com>"
+       },
+       {
+         "name": "Date",
+         "value": "Wed, 7 Oct 2015 12:34:56 -0700"
+       },
+       {
+         "name": "Message-ID",
+         "value": "<0123456789example.com>"
+      },
+      {
+         "name": "Subject",
+         "value": "Test Subject"
+      },
+      {
+        "name": "To",
+        value": "johndoe@example.com"
+      },
+      {
+        "name": "Content-Type",
+        "value": "text/plain; charset=UTF-8"
+      }
+      ],
+        "headersTruncated": false,
+        "messageId": "o3vrnil0e2ic28trm7dfhrc2v0clambda4nbp0g1x"
+      },
+        "receipt": {
+        "recipients": [
+        "johndoe@example.com"
+        ],
+       "timestamp": "1970-01-01T00:00:00.000Z",
+       "spamVerdict": {
+       "status": "PASS"
+      },
+      "dkimVerdict": {
+      "status": "PASS"
+      },
+      "processingTimeMillis": 574,
+      "action": {
+      "type": "Lambda",
+      "invocationType": "Event",
+      "functionArn": functionarn
+      },
+      "spfVerdict": {
+      "status": "PASS"
+      },
+      "virusVerdict": {
+      "status": "PASS"
+      }
+      }
+      },
+      "eventSource": "aws:ses"
+      }
+      ]
+      }]}`
+
+var cloudWatchLogsEvent = `{
+ 	"awslogs": {
+ 		"data": "{{.Data}}"
+ 	}
+ }`
+
+var cognitoSyncTriggerEvent = `{
+  "datasetName": "datasetName",
+  "eventType": "SyncTrigger",
+  "region": "{{.Region}}",
+  "identityId": "identityId",
+  "datasetRecords": {
+    "SampleKey2": {
+      "newValue": "newValue2",
+      "oldValue": "oldValue2",
+      "op": "replace"
+    },
+    "SampleKey1": {
+      "newValue": "newValue1",
+      "oldValue": "oldValue1",
+      "op": "replace"
+    }
+  },
+  "identityPoolId": "identityPoolId",
+  "version": 2
+}`
+
+var lexEvent = `{
+  "messageVersion": "1.0",
+  "invocationSource": "FulfillmentCodeHook or DialogCodeHook",
+  "userId": "user-id specified in the POST request to Amazon Lex.",
+  "sessionAttributes": {
+     "key1": "value1",
+     "key2": "value2",
+  },
+  "bot": {
+    "name": "bot-name",
+    "alias": "bot-alias",
+    "version": "bot-version"
+  },
+  "outputDialogMode": "Text or Voice, based on ContentType request header in runtime API request",
+  "currentIntent": {
+    "name": "intent-name",
+    "slots": {
+      "slot-name": "value",
+      "slot-name": "value",
+      "slot-name": "value"
+    },
+    "confirmationStatus": "None, Confirmed, or Denied
+      (intent confirmation, if configured)"
+  }
+}`
+
+var mobileBackendEvent = `{
```

Comment:
what is a mobile backend? Which AWS service generates it?

Reply:
The documentation isn't too clear but to my knowledge it's generated by AWS Mobile: http://docs.aws.amazon.com/mobile/sdkforandroid/developerguide/getting-started-lambda.html

the event itself is from here:
http://docs.aws.amazon.com/lambda/latest/dg/eventsources.html#eventsources-mobile-backend

Reply:
Okay. Let's not do this because it can be quickly confused with API Backends. The event structure is simple enough that anyone can manually write. Can you remove it?

Reply:
Yup, i'll remove it